scss/05-utilities/_utilities.scss

```diff
@@ -59,109 +108,157 @@
   text-decoration-skip: ink;
 }
 
-// Width
+
+
+
+
+/* Object controls
+ *============================================================================*/
+
+.obj-cover   { object-fit: cover; }
+.obj-contain { object-fit: contain; }
+
+
+
+
+
+/* Position, width, height
+ *============================================================================*/
+
+.relative { position: relative; }
+
 .w-full { width: 100%; }
+.w-auto { width: auto; }
 
-// Height
 .h-full { height: 100%; }
+.h-auto { height: auto; }
+
+
+
+
+
+/* Hide & show
+ *============================================================================*/
+
+.hide-xs { @media (max-width: $screen-xs) { display: none; } }
+.hide-s  { @media (max-width: $screen-s)  { display: none; } }
+.hide-m  { @media (max-width: $screen-m)  { display: none; } }
+.hide-l  { @media (max-width: $screen-l)  { display: none; } }
+
+.show-xs { display: none; @media (max-width: $screen-xs) { display: inline-block; } }
+.show-s  { display: none; @media (max-width: $screen-s)  { display: inline-block; } }
+.show-m  { display: none; @media (max-width: $screen-m)  { display: inline-block; } }
+.show-m  { display: none; @media (max-width: $screen-l)  { display: inline-block; } }
 
-// Text align
-.ta-left    { text-align: left; }
-.ta-center  { text-align: center; }
-.ta-right   { text-align: right; }
-.ta-justify { text-align: justify; }
 
-// Hide & show
-.hide-xs { @media (max-width: 1200px) { display: none; } }
-.hide-s  { @media (max-width: 800px) { display: none; } }
-.hide-m  { @media (max-width: 450px) { display: none; } }
-.show-xs { display: none; @media (max-width: 1200px) { display: inline-block; } }
-.show-s  { display: none; @media (max-width: 800px) { display: inline-block; } }
-.show-m  { display: none; @media (max-width: 450px) { display: inline-block; } }
 
-// Paddings
-.p-a-0 { padding-top: 0; padding-right: 0; padding-bottom: 0; padding-left: 0; }
+
+
+/* Paddings
+ *============================================================================*/
+
+.p-a-0   { padding-top: 0; padding-right: 0; padding-bottom: 0; padding-left: 0; }
+
 .p-a-xs  { @include fluid(10px, 5px, padding); }
```

Comment:
We have `xxl` but we don't have `xxs` what happens with people with OCD who see this? You ignorant bastard.

Reply:
Should we also have an option to add paddings/margins that aren't fluid?